goinstant/prompts.go

```diff
@@ -28,6 +28,7 @@ func selectSetup() error {
 	if !cfg.DisableIG {
 		items = append(items[:index+1], items[index:]...)
 		items[index] = "Install FHIR package"
+		index++
```

Comment:
I originally removed this because it was completely unnecessary computation taking place. Is there a reason you added it other than trying to future-proof it for the next developer(if we ever need to add another feature flag here)?

Reply:
Removed